marklogic-data-hub-central/ui/e2e/cypress/integration/monitor/monitorTableValidations.spec.tsx

```diff
@@ -94,6 +100,17 @@ describe("Monitor Tile", () => {
     });
     monitorPage.getTableNestedRows().should("be.visible");
 
+    cy.log("**check step datetime order DESC by default**");
+    cy.get(".reset-expansion-style:eq(" + monitorPage.searchBiggerRowIndex(firstPageTableCellsJobId) + ") .stepStartDate").then(($row) => {
```

Comment:
Can we move the cy.get's to a page object class? And just those methods from the test